javascript/javascript2/week1/homework/spirit-animal.js

```diff
@@ -0,0 +1,44 @@
+const spiritAnimals = [
+  "The lazy panda",
+  "The running rabbit",
+  "The wise owl",
+  "The fullmoon wolf",
+  "The crying butterfly",
+  "The king lion",
+  "The cunning fox",
+  "The busy bee",
+  "The fluffy cat",
+  "The black sheep",
+];
+
+//const answer = document.createElement("p");
+const answer = document.getElementById("answer");
+const inputField = document.getElementById("username");
+const myButton = document.getElementById("mybutton");
+const radioByClick = document.getElementById("byclick");
+const radioByHoverOver = document.getElementById("byhoverover");
+const radioByFillOut = document.getElementById("byfillout");
+
+myButton.addEventListener("click", myFunction);
+function myFunction() {
+    const username = document.getElementById("username").value.trim();
+    if (username === "" || username === undefined) {
```

Comment:
Great stuff! 

One thing to keep in mind - it's not directly relevant here, but once we code against APIs it will be - is that username could theoretically also be `null`, and in that case this check won't work. A way to check for both `""`, `undefined` and `null` would be `if (!username)`

Reply:
Thanks! All the time I had a feeling that there is another way to do this which includes all possible conditions😅